### PR TITLE
ekf2: Split static pressure correction into separate RH and LH ellipsoids

### PR DESCRIPTION
**Pull request by priseborough**
Splits ellipsoid used to correct for static source position error into separate 
RH and LH regions. This is required where a pixhawk 'cube' autopilot is externally mounted with the usb port facing left or right causing a rise or fall in internal pressure with RH vs LH movement.

### CODE REVIEW

**Comment by priseborough** [comment]
@thomasgubler FYI

**Comment by lamping7** [comment]
Don't get mad at Jenkins... Improvements to the HITL testing platform suite by @dagar made a change in the Jenkinsfile... Rebase.

**Comment by thomasgubler** [comment]
diff looks good to me

**Comment by thomasgubler** [comment]
@bresch

File: src/modules/ekf2/ekf2_main.cpp [hunk 1]
```diff
@@ -485,8 +485,10 @@ class Ekf2 final : public ModuleBase<Ekf2>, public ModuleParams
 		_K_pstatic_coef_xp,	///< static pressure position error coefficient along the positive X body axis
 		(ParamFloat<px4::params::EKF2_PCOEF_XN>)
 		_K_pstatic_coef_xn,	///< static pressure position error coefficient along the negative X body axis
-		(ParamFloat<px4::params::EKF2_PCOEF_Y>)
-		_K_pstatic_coef_y,	///< static pressure position error coefficient along the Y body axis
+		(ParamFloat<px4::params::EKF2_PCOEF_YP>)
+		_K_pstatic_coef_yp,	///< static pressure position error coefficient along the positive Y body axis
+		(ParamFloat<px4::params::EKF2_PCOEF_YN>)
+		_K_pstatic_coef_yn,	///< static pressure position error coefficient along the negativeY body axis
```
Comment (bkueng) [inline src/modules/ekf2/ekf2_main.cpp]:
```suggestion
		_K_pstatic_coef_yn,	///< static pressure position error coefficient along the negative Y body axis
```

File: src/modules/ekf2/ekf2_params.c [hunk 2]
```diff
@@ -1203,15 +1203,27 @@ PARAM_DEFINE_FLOAT(EKF2_PCOEF_XN, 0.0f);
 
 /**
  * Pressure position error coefficient for the Y axis.
```
Comment (bkueng) [inline src/modules/ekf2/ekf2_params.c]: The title description should also be different between the 2 parameters, so e.g. `Pressure position error coefficient for the positive Y axis.`

File: src/modules/ekf2/ekf2_main.cpp (hunk 1 above)
Comment (priseborough) [reply src/modules/ekf2/ekf2_main.cpp]: Fixed

File: src/modules/ekf2/ekf2_params.c (hunk 2 above)
Comment (priseborough) [reply src/modules/ekf2/ekf2_params.c]: Fixed

**Pull request merged**